src/components/ground-nav/ground-nav.twig

```diff
@@ -1,92 +1,172 @@
-{% set kudos %}
-  {% block kudos %}{% endblock %}
+{% set feature_one %}
+  <div class="c-ground-nav__feature-inner o-rhythm o-rhythm--condensed">
+    <div class="o-rhythm o-rhythm--compact">
+      {% embed '@cloudfour/components/heading/heading.twig' with {
+        "tag_name": "h2",
+        "level": 1,
+        "weight": "light"
+      } only %}
+        {% block content %}
+          Nice to meet you
+        {% endblock %}
+      {% endembed %}
+      <p>
+        Cloud Four helps organizations solve complex responsive web
+        design and development challenges every day.
+        Let's connect so we can tailor a solution to fit your needs.
+      </p>
+    </div>
+    {% embed '@cloudfour/objects/button-group/button-group.twig' only %}
+      {% block content %}
+        {% include '@cloudfour/components/button/button.twig' with {
+          label: 'See our work',
+          class: 'c-button--secondary'
+        } only %}
```

Comment:
Sorry, just noticed something. This renders as a `button`. I expected this to be an anchor `a` link instead. Does that matter for this PR?

Reply:
Same with "Get in touch", I expected that to be an `a`, not a `button`.

Reply:
Does not matter for this PR — the contents of the features are placeholders for demo purposes.